### PR TITLE
Transfer tools from ASaiM repository 

### PR DESCRIPTION
**Pull request by gallardoalba**
Only the tools which are currently installed have been moved. Related to https://github.com/ASaiM/galaxytools/pull/47

### CODE REVIEW

**Comment by bebatut** [comment]
Thanks so much @gallardoalba!!!

File: tools/combine_metaphlan2_humann2/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Metagenomics 
+description: Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances
+name: combine_metaphlan2_humann2 
+owner: bebatut
```
Comment (bgruening) [inline tools/combine_metaphlan2_humann2/.shed.yml:5]: @bebatut we would need to have admin rights for those repos.

Comment (bebatut) [reply tools/combine_metaphlan2_humann2/.shed.yml:5]: we can change there the ownership.... happy to transfer the ownership

**Comment by gallardoalba** [comment]
I'm trying to fix the problem linked to `dirname`, but I think it is related to incompatibilities of **group_humann2_uniref_abundances_to_go** with Python3.

File: tools/combine_metaphlan2_humann2/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Metagenomics 
+description: Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances
+name: combine_metaphlan2_humann2 
+owner: bgruening
```
Comment (bebatut) [inline tools/combine_metaphlan2_humann2/.shed.yml:5]:
```suggestion
owner: bebatut
```

File: tools/format_cd_hit_output/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Fasta Manipulation
+description: Format CD-hit output to rename representative sequences with cluster name and/or extract distribution inside clusters given a mapping file
+name: format_cd_hit_output
+owner: bgruening
```
Comment (bebatut) [inline tools/format_cd_hit_output/.shed.yml:5]:
```suggestion
owner: bebatut
```

File: tools/compare_humann2_output/.shed.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Metagenomics 
+description: Compare outputs of HUMAnN2 for several samples and extract similar and specific information
+name: compare_humann2_output
+owner: bgruening
```
Comment (bebatut) [inline tools/compare_humann2_output/.shed.yml:5]:
```suggestion
owner: bebatut
```

File: tools/format_metaphlan2_output/.shed.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Metagenomics
+description: Format MetaPhlAn2 output to extract abundance at different taxonomic levels
+name: format_metaphlan2_output
+owner: bgruening
```
Comment (bebatut) [inline tools/format_metaphlan2_output/.shed.yml:5]:
```suggestion
owner: bebatut
```

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Let's see if this works. Can someone check if those tools are on the auto-install list on EU .. if so the update should happen automtaically over the weekend.

**Comment by gallardoalba** [comment]
Asaim tools are not in the auto-install list on EU, so I guess I need to include them in the bgruening.yaml file.

**Comment by bgruening** [comment]
Yes, that would be nice.

**Comment by bebatut** [comment]
Thanks @gallardoalba for the work there

**Comment by bernt-matthias** [comment]
Note, this has not been deployed: https://github.com/bgruening/galaxytools/actions/runs/3282394110

Since humann2 and metaphlan2 have been deprecated at IUC (in favor of humann and methaphlan in version 3) I was wondering if we deprecate these tools as well. If they still work with version 3 outputs we could/should keep them (but maybe change the name)

**Pull request merged**